vendor/vendor.json

```diff
@@ -1,6 +1,6 @@
 {
 	"comment": "",
-	"ignore": "test",
+	"ignore": "appengine test github.com/hashicorp/nomad/ github.com/hashicorp/terraform/backend",
```

Comment:
Where are these these additional ignores coming from and why do we need them? I can't find any references of them in our current code.

Reply:
They might be a leftover from a manual import that I then aborted.
I see that they don't affect the rest of the vendor directory.
I will remove them at my next import (Terraform upstream code)